Back End/MemorizeWords/MemorizeWords/Infrastructure/Persistance/Repository/WordAnswerRepository.cs

```diff
@@ -94,7 +95,18 @@ private bool GetGivenAnswer(WordAnswerRequest wordAnswerRequest, out WordEntity
         {
             wordEntity = _dbContext.Word.FirstOrDefault(x => x.Id == wordAnswerRequest.WordId) ?? throw new KeyNotFoundBusinessException($"wordId: {wordAnswerRequest.WordId} couldn't found");
 
-            bool answer = wordEntity.Meaning.ToUpperInvariant().Equals(wordAnswerRequest.GivenAnswerMeaning.ToUpperInvariant().ToUpper(), StringComparison.OrdinalIgnoreCase);
+            string targetLanguage = wordEntity.Meaning;
+            
+            if ((LanguageType)wordAnswerRequest.AnswerLanguageType == LanguageType.LearningLanguage)
```

Comment:
Make this if else blocks a method. But the name can be GetCrossCheckWord(wordEntity)

Reply:
If you see a code block for only a purpose, always extract method.